docs/interactions.md

```diff
@@ -1,310 +1,372 @@
+                    L2.RealityETH.cancelArbitration()
 
-    Charlie L1  ForkManager.deployFork(true, contested question data)
-                # Clones ForkManager
-                # Clones Bridge
-                # Copies contested question to child RealityETH
+    [Bob]   L2.L2ForkArbitrator.claimRefund() # Transfers native token back
 
 ```
 
-Next step:
-* Wait for the fork date, then anyone can [Execute an arbitrator removal](#execute-an-arbitrator-removal) on one chain and [Cancel an arbitrator removal](#cancel-an-arbitrator-removal) on the other.
+#### Next step
+
+ * Wait for the fork date, then anyone can [Execute an arbitrator removal](#execute-an-arbitrator-removal) on one chain and [Cancel an arbitrator removal](#cancel-an-arbitrator-removal) on the other.
 
 
-### Bid in the auction
+### Bid in the auction 
+
+TODO: Build this
+
 ```
-    Bob    L2   IncentivizedMarket.bid(uint256 yes_price_percent, value=tokens)
-                    # burns own tokens
-                    # fork.mint(msg.sender, tokens)
+    Bob    L2.IncentivizedMarket.bid(uint256 yes_price_percent, value=tokens)
+                burns own tokens
+                fork.mint(msg.sender, tokens)
 
-                # Wait 1 week
+                Wait 1 week
                 IncentivizedMarket.calculateClearingPrice()
 
-          L2    .getYesNo()
+           L2.IncentivizedMarket.getYesNo(bob)
                 withdraw()
-                    # check clearing price
-                    # decide which side the user is on by whether their price is above or below the clearing percent
-                    # give them tokens, multiplied by inverse of clearing percent
+                check clearing price
+                decide which side the user is on by whether their price is above or below the clearing percent
+                give them tokens, multiplied by inverse of clearing percent
 
-            * ISSUE: Is there a simple implementation of an auction with incentivized liquidity
+#### Need to build:
 
+ * Move burned funds into two pools on L1
+ * Sell the burned funds in return for token A or token B on a curve
+ * See which has the most tokens unsold, that one is more valuable
+ * eg 10000 F1+F2 split into 10000 F1 and 10000 F2
+ * First F1 sells for 0.01 ETH
+ * Second F1 sells for 0.02 ETH
 
-    Move burned funds into two pools on L1
-    Sell the burned funds in return for token A or token B on a curve
-    See which has the most tokens unsold, that one is more valuable
-      eg 10000 F1+F2 split into 10000 F1 and 10000 F2
-         First F1 sells for 0.01 ETH
-         Second F1 sells for 0.02 ETH
-    
 
 ```
 
-### Propose a routine governance change
+### Propose a contract upgrade
 ```
-    Charlie L1  ForkManager.beginUpgradeBridge
-                    gov_question_id = RealityETH.askQuestion("should we do a routine upgrade to ForkManager XYZ?")
+    Charlie L1.ForkableRollupGovernor.beginProposition
```

Comment:
what was the reason that we don't propose it on L2? Is it because L2 might be broken?

Reply:
Right, exactly. We need it on L2 for the cases where L1 is broken, including if somebody has censorship power (so they could break an L2 escalation game).